Jellyfin.Api/Controllers/RemoteImageController.cs

```diff
@@ -163,7 +170,21 @@ public ActionResult<IEnumerable<ImageProviderInfo>> GetRemoteImageProviders([Fro
             return NotFound();
         }
 
-        await _providerManager.SaveImage(item, imageUrl, type, null, CancellationToken.None)
+        var provider = _providerManager.GetImageProviders(item, new(new DirectoryService(_fileSystem)))
```

Comment:
Way too much responsibility for the controller. The controller should not care about providers. The manager should manage it.